tests/resources/test_syntaxes.py

```diff
@@ -0,0 +1,62 @@
+"""
+    tests.resources.test_syntaxes
+    -----------------------------
+
+    Tests Syntaxes resource.
+
+    :copyright: (c) 2017 The XSnippet Team, see AUTHORS for details
+    :license: MIT, see LICENSE for details
+"""
+
+import pytest
+
+
```

Comment:
Can we also ensure expected behavior when no `syntaxes` set in config? Maybe respond with empty array or so?

Reply:
Ah, good catch! This case wasn't covered...